test/author_test.rb

```diff
@@ -0,0 +1,34 @@
+require "minitest/autorun"
+require "minitest/pride"
+require "./lib/author"
+require "./lib/book"
+
+class AuthorTest < Minitest::Test
+  def setup
+    @charlotte_bronte = Author.new({
+                                    first_name: "Charlotte",
+                                    last_name: "Bronte"
+                                    })
+  end
+
+  def test_it_exists_and_has_attributes
+    assert_instance_of Author, @charlotte_bronte
+  end
+
+  def test_it_can_find_author_name
+    assert_equal "Charlotte Bronte", @charlotte_bronte.name
+  end
+
+  def test_it_can_add_books
+    assert_equal [], @charlotte_bronte.books
+
+    jane_eyre = @charlotte_bronte.write("Jane Eyre", "October 16, 1847")
+
+    assert_equal Book, jane_eyre[0].class
```

Comment:
Careful here. We wanted the return value of the write to be a Book object, if you have to call [0] on it, it means it's an Array.